### ISSUE TITLE
NCOL write wrong attribute type has confusing warning

### ISSUE THREAD

**Issue by GroteGnoom** [igraph/igraph#1941]
**Describe the bug**
If you try to write a graph to NCOL, and you include vertex names or edge weights it expects the names to be strings, and the weights to be numbers. If you try to pass a number as a vertex name, it is ignored and the warning says 'unknown attribute type.'. It seems to me that it should make clear that it expects a string. The attribute type is not unknown. 

**To reproduce**
```
int main() {
    igraph_t g;
    FILE *file;

    igraph_set_attribute_table(&igraph_cattribute_table);

    igraph_small(&g, 4, IGRAPH_DIRECTED,
            0, 1, 0, 1, 0, 1,
            1, 2, 2, 3,
            -1);

    SETEAN(&g, "edge_attr", 0, 10);
    SETVAN(&g, "vertex_attr", 0, 10);

    file = tmpfile();

    igraph_write_graph_ncol(&g, file, "vertex_attr", "edge_attr");
}
```

edit:
output:
```
Warning at src/io/ncol.c:282 : Ignoring names attribute 'vertex_attr', unknown attribute type.
```

**Version information**
131992dc0961744fb9911f336910ee1b0b19c23f

**Comment by szhorvat** [comment]
Can you include the output of the program please?

**Comment by szhorvat** [comment]
If you have not seen it, it's good to be aware of this fix: https://github.com/igraph/igraph/issues/1919

**Comment by GroteGnoom** [comment]
Fixed in 47555023dfb88c14d8d729df5d4eb0511b69153b
